spec/supabase_js_v2.yml

```diff
@@ -1558,6 +1558,8 @@ pages:
     $ref: '@supabase/postgrest-js.PostgrestQueryBuilder.upsert'
     notes: |
       - Primary keys must be included in `values` to use upsert.
+      - If you are using RLS, you might want to add some data to upsert.
+      - If the table is restricted to row `owners`, you need to include the column that the auth.id matches when doing bulk upsert request only when a select number of the columns are changing.
```

Comment:
- I don't think was meant to be copied and pasted: `you might want to add some data to upsert.`
- I think `auth.id` should be `auth.uid` per https://supabase.com/docs/guides/auth/row-level-security#helper-functions
- This should be one bullet point instead of two. Something like "If you're using Row Level Security and the table is restricted to row `owners`, do X, Y, Z." (I don't quite understand what the last part of the sentence is saying.)

cc: @thorwebdev

Reply:
Thank you for the review, I was confused as well while working on it. Working on the requested changes now!